README.md

```diff
@@ -7,6 +7,45 @@
 
 Elixir School's Extracurricular is a website and twitter bot intended to increase visibility to open source opportunities within the community.
 
+## Running the Project
+Once you have cloned the repo and `cd`'d in, its quite simple to get started.
+First, we get our dependencies and compile everything.
+
+```bash
+$ mix do deps.get, compile
+```
+
+Now, you will need to have Postgres running and have a user `postgres` with the password `postgres`, or fill out the proper credentials in `apps/web/config/dev.exs` and `apps/web/config/test.exs`.
+Once you have gotten this proper, its time to create the database and run our migrations.
+
+```bash
+$ mix do ecto.create, ecto.migrate
+```
+
+Now we seed the database
+
+```bash
+$ mix run apps/data/priv/repo/seeds.exs
+```
+
+From here, we need to build all of our javascript pieces.
+
+```bash
+$ cd apps/web/assets
```

Comment:
I'm not sure these are accurate or necessary, are they?  I ran the project earlier and `phx.server` took care of compiling my assets but I think @burden set it up to work with webpack and not brunch.

Reply:
Oh I've never used webpack but when I did a normal run of `phx.server` without having done this it wouldnt work, and brunch wouldnt run until I had `cd`'d here.